subprojects/files/src/main/java/org/gradle/internal/file/impl/DefaultFileMetadata.java

```diff
@@ -20,20 +20,28 @@
 import org.gradle.internal.file.FileType;
 
 public class DefaultFileMetadata implements FileMetadataSnapshot {
```

Comment:
Can we please rename this to `DefaultFileMetadataSnapshot`? I know it's not related, but it's a small step towards making things more consistent.

Reply:
Renamed as part of #13039.